modules/lightning_features/lightning_media/src/Plugin/EntityBrowser/Widget/FileUpload.php

```diff
@@ -65,9 +65,12 @@ public function getForm(array &$original_form, FormStateInterface $form_state, a
     // it as the first validator, allowing it to accept only file extensions
     // associated with existing media bundles.
     if (empty($validators['file_validate_extensions'])) {
+      $entity_browser_info = $form_state->get('entity_browser');
```

Comment:
I think $form_state->get() will take an additional argument (the default value). To prevent potential weirdness, let's do $form_state->get('entity_browser', []) so that we always get an array back.